products/rhel8/profiles/stig.profile

```diff
@@ -856,8 +856,7 @@ selections:
     # RHEL-08-030720
 
     # RHEL-08-030730
-    # this rule expects configuration in MB instead percentage as how STIG demands
-    # - auditd_data_retention_space_left
+    - auditd_data_retention_space_left_percentage
```

Comment:
```suggestion
    - auditd_data_retention_space_left_percentage
    
    # RHEL-08-030731
```

The rule below `auditd_data_retention_space_left_action` has been split a there is a new STIG id assigned to it. The rule.yml itself has been updated accordingly. But the metadata in the profile file has not.

Reply:
@ggbecker Done.